docs/getting_started/tutorial.md

```diff
@@ -563,11 +585,13 @@ Adjust your blog page template to include the images:
 {% endblock %}
 ```
 
+Make sure to upload some images when editing the blog page on your Wagtail admin if you want them to be displayed after editing your blog page template.
+
 Here we use the `{% image %}` tag (which exists in the `wagtailimages_tags` library, imported at the top of the template) to insert an `<img>` element, with a `fill-320x240` parameter to indicate that the image should be resized and cropped to fill a 320x240 rectangle. You can read more about using images in templates in the [docs](../topics/images).
 
 !["Second Post" page, with title, date, intro, body, and a gallery of three images](../_static/images/tutorial/tutorial_6.png)
 
-Since our gallery images are database objects in their own right, we can now query and re-use them independently of the blog post body. Let's define a `main_image` method, which returns the image from the first gallery item (or `None` if no gallery items exist):
+Since our gallery images are database objects in their own right, we can now query and re-use them independently of the blog post body. Let's define a `main_image` method, which returns the image from the first gallery item (or `None` if no gallery items exist) in `blog/models.py`:
```

Comment:
Since this is just the one method specifying where to add it doesn’t seem worthwhile to me? It has to be where `BlogPage` is defined to start with.

```suggestion
Since our gallery images are database objects in their own right, we can now query and re-use them independently of the blog post body. Let's define a `main_image` method, which returns the image from the first gallery item (or `None` if no gallery items exist):
```